pkg/metrics/storage/storage_sampler_linux.go

```diff
@@ -274,6 +274,21 @@ func parseMountInfo(line string) (mi MountInfoStat, err error) {
 	mi.FSType = fields[separator+1]
 	mi.MountSource = fields[separator+2]
 	mi.Device = fields[separator+2]
+
+	if len(fields) >= separator+4 {
+		superOpts := strings.Split(fields[separator+3], ",")
+		if len(superOpts) > 0 {
```

Comment:
I think we can remove this if statement, if no elements, no values would be returned from the range.

Reply:
if I understood this correctly, `fields[separator+3]` could be empty (theoretically)

Reply:
but as for accepting empty slice, I will remove this check 👍